webui/react/src/pages/ModelVersionDetails.tsx

```diff
@@ -186,7 +187,9 @@ const ModelVersionDetails: React.FC = () => {
       .sort((a, b) => checkpointResources[a] - checkpointResources[b])
       .map((key) => ({ name: key, size: humanReadableBytes(checkpointResources[key]) }));
     const hasExperiment = !!modelVersion.checkpoint.experimentId;
-    return [
+    const pachydermData = modelVersion.checkpoint.experimentConfig?.integrations?.pachyderm;
+    const url = pachydermData ? createPachydermLineageLink(pachydermData) : undefined;
```

Comment:
nit: move this inside the if statement -- this prevents `url` from being set to undefined